twingate/internal/utils/utils.go

```diff
@@ -22,3 +23,32 @@ func Filter[T any](items []T, ok func(item T) bool) []T {
 
 	return result
 }
+
+// Contains - checks if element exists in the slice.
+func Contains[T comparable](items []T, element T) bool {
+	lookup := MakeLookupMap[T](items)
```

Comment:
yes, agree it will be pretty the same, the idea was just to re-use existing code

but let's do as you propose, as we don't need to allocate extra memory for this